terragrunt/aws/alarms/billing_alarm.tf

```diff
@@ -12,7 +12,7 @@ resource "aws_cloudwatch_metric_alarm" "billing_change_over_threshold" {
 
   metric_query {
     id          = "anomaly"
-    expression  = "ANOMALY_DETECTION_BAND(current)"
+    expression  = "ANOMALY_DETECTION_BAND(m1, 3)"
```

Comment:
might have to leave this as `current` since there's no reference to `m1` anywhere and you've updated how `current` is generated, so should be ok to leave as-is

Reply:
Ah right, done. Copy and paste casualty!